catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalogOperations.java

```diff
@@ -510,4 +513,16 @@ public PropertiesMetadata topicPropertiesMetadata() throws UnsupportedOperationE
     throw new UnsupportedOperationException(
         "Jdbc catalog doesn't support topic related operations");
   }
+
+  public static void deregisterDriver(Driver driver) throws SQLException {
```

Comment:
"unload" should be more suitable.

Reply:
I noticed that `DriverManager` also uses `deregister`, so are we going to rename it to `upload`?

Reply:
It's fine if driver manager uses `deregister`.

Reply:
👌